solidity/README.md

```diff
@@ -0,0 +1,70 @@
+﻿Solidity contract workflow
+
+Start with source code of the contract in <MyContract.sol>
+
+1) Compile Solidity source to TVM assembler: 
+
+solc --tvm MyContract.sol >MyContract.code
+
+2) Generate public interface for the contract in JSON format: 
+
+solc --tvm_abi MyContract.sol >MyContract.abi.json
+
+3) Assemble and link with libraries of your choice into TVM bytecode: 
+
+tvm_linker compile MyContract.code --lib <path to>stdlib_sol.tvm --abi-json MyContract.abi.json
+
+Binary code of your contract is recorded into <MyContractAddress>.tvc file, where 
+<MyContractAddress> is the address of the contract.
+Contract is ready to be deployed onto local node or blockchain.
```

Comment:
The contract is ready for deployment to a local node, testnet or mainnet.

Reply:
Will be addressed as a part of describing contract deployment.